metis-dereference/metis-dereference-service/src/main/java/eu/europeana/metis/dereference/service/MongoDereferenceService.java

```diff
@@ -81,7 +84,10 @@ public EnrichmentResultList dereference(String resourceId)
     }
 
     // Prepare the result: empty if we didn't find an entity.
-    return new EnrichmentResultList(resultList);
+    final List<EnrichmentBaseWrapper> enrichmentBaseWrapperList = resultList.stream()
```

Comment:
These three lines of code occur a lot, mainly in test code. Maybe it's worth making this into a static conversion method? And perhaps put it in EnrichmentBaseWrapper?

Reply:
Fixed.